### ISSUE TITLE
Error scanning using github action with 2.19.0

### ISSUE THREAD

**Issue by trdyer** [jfrog/frogbot#572]
### Describe the bug

After updating to v2.19.0 of the github action all scans fail.

```
Run jfrog/frogbot@v2
  with:
    version: latest
  env:
    JF_URL: https://<redacted>.jfrog.io
    JF_ACCESS_TOKEN: ***
    JF_GIT_TOKEN: ***
    JF_GIT_BASE_BRANCH: main
Frogbot
  /opt/hostedtoolcache/frogbot/[RELEASE]/x64/frogbot scan-repository
  [14](https://github.com/<REPONAME>/actions/runs/<some number>/job/<other number>#step:2:14):45:46 [Info] Frogbot version: 2.19.0
  14:45:46 [Info] Running Frogbot "scan-repository" command
  14:45:48 [Info] Preforming 1 SCA scans:
  [
    {
      "Technology": "go",
      "WorkingDirectory": "/tmp/jfrog.cli.temp.-[16](https://github.com/<REPONAME>/actions/runs/<some number>/job/<other number>#step:2:16)99541146-837276097",
      "Descriptors": [
        "/tmp/jfrog.cli.temp.-1699541146-837[27](https://github.com/<REPONAME>/actions/runs/<some number>/job/<other number>#step:2:28)6097/go.mod"
      ]
    }
  ]
  14:45:48 [Info] Running SCA scan for go vulnerable dependencies in /tmp/jfrog.cli.temp.-1699541146-837276097 directory...
  14:45:48 [Info] Calculating Go dependencies...
  Error: 8 [Error] audit command in '/tmp/jfrog.cli.temp.-1699541146-837276097' failed:
  failed while building 'go' dependency tree:
  failed while reading go.yaml config file: Server ID '<server name from config that *does* exist>' does not exist.
  Please run 'jf go-config' with your resolver repository information
  Error: The process '/opt/hostedtoolcache/frogbot/[RELEASE]/x64/frogbot' failed with exit code 1
```

Re-running an action from Tuesday that successfully completed with 2.18.0 will now use 2.19.0 and fail.

### Current behavior
```
Run jfrog/frogbot@11e0dc90fafcf7e447c4c77c50e2abfdf87799e2 #v2.17.1
  with:
    version: latest
  env:
    JF_URL: https://<redacted>.jfrog.io
    JF_ACCESS_TOKEN: ***
    JF_GIT_TOKEN: ***
    JF_GIT_BASE_BRANCH: main
    JFROG_CLI_LOG_LEVEL: DEBUG
Frogbot
  /opt/hostedtoolcache/frogbot/[RELEASE]/x64/frogbot scan-repository
  14:50:54 [Info] Frogbot version: 2.19.0
  14:50:54 [Debug] Reading config from file system. Looking for .frogbot/frogbot-config.yml
  14:50:54 [Debug] frogbot-config.yml wasn't found in /home/runner/work/<projectname>/.frogbot/frogbot-config.yml. Searching for it in upstream directories
  14:50:54 [Debug] Sending HTTP HEAD request to: 'https://github.com/jfrog/frogbot'
  14:50:54 [Debug] Locking config file to run config AddOrEdit command.
  14:50:54 [Debug] Creating lock in: /tmp/jfrog.cli.temp.-1699541454-3467646434/locks/config
  14:50:54 [Debug] Releasing lock: /tmp/jfrog.cli.temp.-1699541454-3467646434/locks/config/jfrog-cli.conf.lck.1677.1699541454436308672
  14:50:54 [Debug] Config AddOrEdit command completed successfully. config file is released.
  14:50:54 [Debug] Usage Report: Sending info...
  14:50:54 [Debug] Sending HTTP POST request to: https://usage-ecosystem.jfrog.io/api/usage/report
  14:50:54 [Info] Running Frogbot "scan-repository" command
  14:50:54 [Debug] Sending HTTP GET request to: https://<redacted>.jfrog.io/artifactory/api/system/version
  14:50:55 [Debug] Setting timeout for go-git to 120 seconds ...
  14:50:55 [Debug] Created temp working directory: /tmp/jfrog.cli.temp.-1699541455-1611871835
  14:50:55 [Debug] Cloning <https://github.com/<REPONAME>.git/origin/refs/heads/main>...
  14:50:56 [Debug] Project cloned from https://github.com/<REPONAME>.git to /tmp/jfrog.cli.temp.-1699541455-1611871835
  14:50:56 [Info] Executing 'go mod download' at /tmp/jfrog.cli.temp.-1699541455-1611871835
  14:51:10 [Debug] Sending HTTP GET request to: https://<redacted>.jfrog.io/xray/api/v1/system/version
  14:51:10 [Debug] Sending HTTP GET request to: https://<redacted>.jfrog.io/xray/api/v1/entitlements/feature/contextual_analysis
  14:51:11 [Debug] The path '/tmp/jfrog.cli.temp.-1699541455-1611871835/.github/workflows/migration-tests.yml' is excluded
...

Source files are being marked as excluded

...
  14:51:11 [Debug] mapped 1 working directories with indicators/descriptors:
  {
    "/tmp/jfrog.cli.temp.-1699541455-1611871835": [
      "/tmp/jfrog.cli.temp.-1699541455-1611871835/go.mod"
    ]
  }
  14:51:11 [Debug] Detected 1 technologies at /tmp/jfrog.cli.temp.-1699541455-1611871835: [go].
  14:51:11 [Info] Preforming 1 SCA scans:
  [
    {
      "Technology": "go",
      "WorkingDirectory": "/tmp/jfrog.cli.temp.-1699541455-1611871835",
      "Descriptors": [
        "/tmp/jfrog.cli.temp.-1699541455-1611871835/go.mod"
      ]
    }
  ]
  14:51:11 [Info] Running SCA scan for go vulnerable dependencies in /tmp/jfrog.cli.temp.-1699541455-1611871835 directory...
  14:51:11 [Info] Calculating Go dependencies...
  14:51:11 [Debug] Preparing to read the config file /tmp/jfrog.cli.temp.-1699541455-1611871835/.jfrog/projects/go.yaml
  14:51:11 [Debug] Found resolver in the config file /tmp/jfrog.cli.temp.-1699541455-1611871835/.jfrog/projects/go.yaml
  Error: 1 [Error] audit command in '/tmp/jfrog.cli.temp.-1699541455-1611871835' failed:
  failed while building 'go' dependency tree:
  failed while reading go.yaml config file: Server ID '<servername>' does not exist.
  Please run 'jf go-config' with your resolver repository information
  Error: The process '/opt/hostedtoolcache/frogbot/[RELEASE]/x64/frogbot' failed with exit code 1
```
### Reproduction steps

Standard scan repository action of a go project

### Expected behavior

Scan to continue working

### JFrog Frogbot version

2.19.0

### Package manager info

go

### Git provider

GitHub

### JFrog Frogbot configuration yaml file

- params:
    git:
      repoName: <reponame>
      branches:
        - main
        - release/*
    scan:
      includeAllVulnerabilities: true
      projects:
        - workingDirs:
            - "."

### Operating system type and version

ubuntu latest 

### JFrog Xray version

Cloud

**Comment by omerzi** [comment]
Hi @trdyer, thank you for reporting this issue. I believe I have identified the cause, which stems from some changes we made to our `jfrog-cli-core` component in the latest release. I have opened a pull request (https://github.com/jfrog/jfrog-cli-core/pull/1024) that I believe addresses this issue, and I will keep you informed once it is merged and released.

**Comment by omerzi** [comment]
@trdyer, we just released Frogbot v2.19.1 with the updated component. Let me know if it resolves your issue. Thanks.

**Comment by trdyer** [comment]
They are now building! Thank you so much!
